tests/ui/crashes/ice-5207.rs

```diff
@@ -0,0 +1,7 @@
+// edition:2018
```

Comment:
commpiletest-rs has an `edition` flag now? `s/compile-flags:\s*--edition[=\s]+2018/edition: 2018/`

Reply:
Yeah, they can accept it: https://github.com/laumann/compiletest-rs/blob/66f68a72a42b01e3d54db8ca3c62acaa91d13cba/src/header.rs#L300-L302

Reply:
But it's space-sensitive, so it should be `edition:2018`, not `edition: 2018` :S